### PR TITLE
Add jquery-localize@0.1.0 w/ npm auto-update

### LINKED ISSUE #9563: [Request] jquery.localize.js
**Library name: Jquery Localize
**Git repository url: https://github.com/coderifous/jquery-localize
**npm package url(optional):** https://www.npmjs.com/package/jquery-localize
**License(s): **GPL** and **MIT**
**Official homepage: https://github.com/coderifous/jquery-localize
**Wanna say something? Leave message here: It's amazing, please add it!

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by NiniHsu**
Pull request for issue: #9563 
Related issue(s): # #9414 
@kennynaoh Please help me review this PR, thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/coderifous/jquery-localize
 * Official website (optional, not the repository): https://github.com/coderifous/jquery-localize
 * NPM package url (optional): https://www.npmjs.com/package/jquery-localize
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 336
   - Count of watchers: 21
   - Count of forks: 98
 * NPM download stats (optional):
   - Downloads in the last day: 3
   - Downloads in the last week: 13
   - Downloads in the last month: 69

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9563, cc #9414, cc @wbinnssmith, @coderifous

### CODE REVIEW

File: ajax/libs/jquery-localize/0.1.0/jquery.localize.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
+
+/*
+Copyright (c) Jim Garvin (http://github.com/coderifous), 2008.
+Dual licensed under the GPL (http://dev.jquery.com/browser/trunk/jquery/GPL-LICENSE.txt) and MIT (http://dev.jquery.com/browser/trunk/jquery/MIT-LICENSE.txt) licenses.
```
Comment (kennynaoh) [inline ajax/libs/jquery-localize/0.1.0/jquery.localize.js]: @NiniHsu Dual license

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
@NiniHsu Thank you for your contribution.
Would you like to finish this PR?
I think the licenses about this lib are clear. Because this lib said it follows jQuery's license. Because it's copyright 2008 so as https://www.openfoundry.org/enterprise-application/8680 said it's under MIT and GPL-2.0.

So just need to modify as the followings:
```js
  "licenses": [
    "MIT",
    "GPL-2.0"
  ],
```
If there is no response in 7 days, I will add this lib ASAP.

Thank you.

**Comment by sufuf3** [comment]
@PeterDaveHello could we close this PR and let intern4 to add this lib in practice?
Because this PR is related #9414.

Thanks.

**Comment by PeterDaveHello** [comment]
Let's close it.

**Pull request closed without merging**